src/commands/misc/itunes.js

```diff
@@ -0,0 +1,60 @@
+const { SwitchbladeEmbed, Command, Constants } = require('../../')
+
+const MEDIA_WHITE_LIST = ['movie', 'podcast', 'music', 'musicVideo', 'audiobook', 'shortFilm', 'tvShow', 'software', 'ebook', 'all']
+
+module.exports = class Itunes extends Command {
+  constructor (client) {
+    super({
+      name: 'itunes',
+      requirements: { apis: ['itunes'] },
+      embedLogoURL: 'https://i.imgur.com/U4jjk5F.png',
+      parameters: [{
+        type: 'string',
+        full: false,
+        name: 'media',
+        whitelist: MEDIA_WHITE_LIST,
+        missingError: 'commands:searchplate.notFound',
```

Comment:
Wrong command

Reply:
What do you mean? In the comment, it was told to use this command

Reply:
@Doges was just giving an example of how you can do it, you weren't supposed to use the same string!

Reply:
Oh, my bad :((